common/middleware/set-locations.js

```diff
@@ -8,16 +8,6 @@ const pipe = functions => data => {
   }, data)
 }
 
-const getParamsLocation = data => {
```

Comment:
I feel like there are some tests associated with this that might need cleaning up?

But maybe it doesn't cater for the way express uses this functionality.

Reply:
Yeah, I forgot to remove the tests for this, in this branch.

It's definitely not used anywhere, as I said, I put a `throw new Error` and ran the tests, and clicked around - didn't see anything.

The only way this could work, is if we set an `app` route of `.*/location/REGEX/.*` and then had every other router inherit params. The addition of location is important, as that would allow us to target the regex specifically - otherwise `allocation` or `population` ids would match as well.